spec.html

```diff
@@ -26957,8 +26957,8 @@ <h1>Example Cyclic Module Record Graphs</h1>
               </tr>
               <tr>
                 <th>_C_</th>
-                <td>2</td>
-                <td>1</td>
+                <td>3</td>
+                <td>0</td>
```

Comment:
Could you explain the [[DFSAncestorIndex]] change? Can't follow the algorithm very well.

Reply:
I believe this is just a typo since it is `0` in all previous cases. The mutable state changes are invariant of the global numbering of DFSIndicies, so it should at least be consistent with the others since it's the same graph as the other tables.